src/main/java/net/honux/chess/attribute/Color.java

```diff
@@ -1,5 +1,5 @@
 package net.honux.chess.attribute;
 
 public enum Color {
-    WHITE, BLACK
+    WHITE, BLACK, NOCOLOR;
```

Comment:
`NOCOLOR` 라는 접근법 좋은 것 같습니다. 
그런데 `enum`은 한 줄에 하나씩 선언해주세요.

```suggestion
        WHITE,
        BLACK,
        NOCOLOR;

Reply:
감사합니다. 혹시 가독성때문인가요?
아니면 관례같은 것인가요?

Reply:
오우 해보니 훨씬더 깔끔해보이기는 하네요 감사해요 👍